api/ShadowRoot.json

```diff
@@ -242,10 +242,28 @@
             "oculus": "mirror",
             "opera": "mirror",
             "opera_android": "mirror",
-            "safari": {
-              "version_added": "16.4"
-            },
-            "safari_ios": "mirror",
+            "safari": [
+              {
+                "version_added": "16.4"
+              },
+              {
+                "prefix": "webkit",
+                "version_added": "10"
```

Comment:
Is this a copy-paste mistake? It's a change of the Safari data (not mentioned in the PR description) and `webkitFullscreenElement` doesn't show up in the WebKit history:
https://github.com/WebKit/WebKit/blob/9af6b6771f33582f16c25548dfa26c6ae11ae990/Source/WebCore/dom/DocumentOrShadowRoot%2BFullscreen.idl

Reply:
Yeah, this would be a copy-paste error.  Reverting.